composer.json

```diff
@@ -18,6 +18,9 @@
   "require-dev": {
     "friendsofphp/php-cs-fixer": "^3.11",
     "phpspec/prophecy-phpunit": "^2.0",
+    "phpstan/phpstan": "^1.8",
+    "phpstan/phpstan-phpunit": "^1.1",
+    "phpstan/phpstan-symfony": "^1.2",
```

Comment:
Should we add `jangregor/phpstan-prophecy`, too?

Reply:
Jup. i can do it. which version?

Reply:
Latest, I'd say, it's a dev dependency.